rtf_vis_tool/src/Components/Node.js

```diff
@@ -0,0 +1,33 @@
+/* Basic Div Component repeatedly rendered within LandingPageGraph.js. It is a div (a rectangular 
+container to hold text) which serves as a node/vertex in a directed graph.
+
+Author: Adi Singh
+*/
+import React from "react";
+
+// Local Imports.
+import '.././stylesheets/Node.css';
+
+const Node = (props) => {
```

Comment:
@dellaert maybe this is too minor, but maybe we should use `GtsfmNode` instead of `Node`? Since "Node" already has a strong meaning in Javascript?

Reply:
I agree. It may be a bit misleading to name a node `Node.js` since it already has a strong meaning in the JS world. Will rename it to `GtsfmNode`